.github/workflows/pr-tests.yaml

```diff
@@ -155,3 +159,86 @@ jobs:
           cd ${GITHUB_WORKSPACE}/unit-tests/build
           export DYNINSTAPI_RT_LIB=$(find ${GITHUB_WORKSPACE}/install/ -name libdyninstAPI_RT.so)
           ctest -VV --output-on-failure .
+
+  usr-tests:
```

Comment:
Instead of duplicating the test job, let's just add the install directory to the existing matrix. Building over the various OSes would be good since they all have different versions of CMake.

```
install-dir: ["${GITHUB_WORKSPACE}/install", "/usr"]
```
and then replace all of the uses of `${GITHUB_WORKSPACE}/install` with `${{ matrix.install-dir }}`.

Reply:
Done.